assets/sass/vendor/_mdc-button.scss

```diff
@@ -87,4 +95,19 @@
 	.mdc-button__trailing-icon {
 		margin-left: 8px;
 	}
+
+	&.googlesitekit-button-icon {
+		min-height: auto;
+		min-width: auto;
+		padding: 8px;
+	}
+
+	&.googlesitekit-button-icon--phone {
```

Comment:
Let's not mix these style definitions in the vendor stylesheets which are more for implementing/adapting Material styles. These probably belong in a new `assets/sass/components/global/_googlesitekit-button-icon.scss` stylesheet.